src/cls/rgw/cls_rgw_ops.h

```diff
@@ -1122,24 +1152,38 @@ struct cls_rgw_lc_list_entries_op {
 WRITE_CLASS_ENCODER(cls_rgw_lc_list_entries_op)
 
 struct cls_rgw_lc_list_entries_ret {
-  map<string, int> entries;
+private:        
+  mutable map<string, int> compat_entries;
```

Comment:
same here, compat_entries only needed for encode/decode, so just have this as a local variable then